### ISSUE TITLE
Custom Object detector for another domain

### ISSUE THREAD

**Issue by akshu281** [quantumjot/BayesianTracker#4]
Hello Author,

Could you please suggest if this Bayesian tracker can be implemented for another domain along with any deep learning based object detector. 

If yes, can you please let me know if the input format of the object detection that has to be followed for running this tracking algorithm ?

Thank You !

**Comment by quantumjot** [comment]
Should work yes, there are details of the input format in the wiki

**Comment by akshu281** [comment]
Thank You. I have taken a look at that. Suppose I am using the JSON based inputloader as given below,

{
  "Object_1": {
    "x": 554.29737483861709,
    "y": 1199.362071438818,
    "z": 0.0,
    "t": 862,
    "label": 0,
    "states": 1,
    "dummy": false
  }
}

Here if I use the custom object detector, I just have to feed x and y to be the center of the bounding box and leave the z as is ? The label, states and dummy attributes should be altered or can be still retained the same way ?

**Comment by quantumjot** [comment]
Yes, that should work. Label, states and dummy can be left as in your example.
